{"name": "test for wait more than 30 sec", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: net::ERR_CONNECTION_REFUSED at http://127.0.0.1:8000/\n=========================== logs ===========================\nnavigating to \"http://127.0.0.1:8000/\", waiting until \"load\"\n============================================================\nNote: use DEBUG=pw:api environment variable and rerun to capture Playwright logs.", "trace": "get_browser = <playwright.sync_api._generated.Browser object at 0x0000028DC60AC4F0>\nrequest = <SubRequest 'desktop_app' for <Function test_wait_more_30_sec>>\n\n    @fixture(scope='session')\n    def desktop_app(get_browser, request):\n        # base_url = request.config.getoption('--base_url') # config 2\n        base_url = request.config.getini('base_url')  # config 3\n        # base_url = request.config.getoption('--base_url')  # config 4\n        # app = App(get_playwright, base_url=settings.BASE_URL) # config 1\n        app = App(get_browser, base_url=base_url, **BROWSER_OPTIONS)  # config 3, 4\n>       app.goto('/')\n\n..\\conftest.py:115: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\page_objects\\application.py:32: in goto\n    self.page.goto(self.base_url + endpoint)\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\playwright\\sync_api\\_generated.py:6714: in goto\n    raise e\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\playwright\\sync_api\\_generated.py:6704: in goto\n    self._sync(\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\playwright\\_impl\\_sync_base.py:90: in _sync\n    return future.result()\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\playwright\\_impl\\_page.py:470: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\playwright\\_impl\\_frame.py:115: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\playwright\\_impl\\_connection.py:36: in send\n    return await self.inner_send(method, params, False)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <playwright._impl._connection.Channel object at 0x0000028DC60ACCA0>\nmethod = 'goto', params = {'url': 'http://127.0.0.1:8000/'}\nreturn_as_dict = False\n\n    async def inner_send(\n        self, method: str, params: Optional[Dict], return_as_dict: bool\n    ) -> Any:\n        if params is None:\n            params = {}\n        callback = self._connection._send_message_to_server(self._guid, method, params)\n>       result = await callback.future\nE       playwright._impl._api_types.Error: net::ERR_CONNECTION_REFUSED at http://127.0.0.1:8000/\nE       =========================== logs ===========================\nE       navigating to \"http://127.0.0.1:8000/\", waiting until \"load\"\nE       ============================================================\nE       Note: use DEBUG=pw:api environment variable and rerun to capture Playwright logs.\n\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\playwright\\_impl\\_connection.py:47: Error"}, "start": 1649695900850, "stop": 1649695900850, "uuid": "0da8dc3b-572b-40e5-bb2a-a0a8b9ad2529", "historyId": "9f6638a2821832bb93087a843ead69f6", "testCaseId": "15b4f072f9916d659c63437fea6195b3", "fullName": "tests.test_wait_page#test_wait_more_30_sec", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_wait_page"}, {"name": "host", "value": "DESKTOP-QSMVGFH"}, {"name": "thread", "value": "11260-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_wait_page"}]}